docs/source/conf.py

```diff
@@ -1060,6 +1060,8 @@
     "reduce_deploy_graph_module",
     "reduce_graph_module",
     "reduce_package_graph_module",
+    # torch.fx.lazy_graph_module
+    "use_lazy_graph_module",
```

Comment:
Adding new entries to this file is not ok.
Please make sure to either document these APIs properly to ensure they show up on the docs website or make them private.

Reply:
I see. I'll make them private (by prepending an underscore) for now unless people want to add documents instead.